src/ledger/LedgerTxn.cpp

```diff
@@ -1526,15 +1461,17 @@ LedgerTxn::Impl::getNewestVersion(InternalLedgerKey const& key,
 
```

Comment:
Need to add an expiration check above in `LedgerTxn::Impl::getNewestVersion` now that `mEntry` may contain expired entries since `load` may make an expired entry active. This also applies in `getNewestVersionEntryMap`.

Reply:
Good point. Fixed